databricks/koalas/series.py

```diff
@@ -321,6 +321,24 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
         kdf._metadata = Metadata(column_fields=[self.name], index_info=[(index_name, None)])
         return _col(kdf)
 
+    @derived_from(pd.Series)
```

Comment:
Since you put documentation already, you do not need this annotation (which is just here to inject the documentation from pandas).

Reply:
You are right! Thanks!